cmake/mcuboot.cmake

```diff
@@ -2,6 +2,34 @@
 set(MCUBOOT_DIR ${ZEPHYR_BASE}/../bootloader/mcuboot)
 
 if(CONFIG_BOOTLOADER_MCUBOOT)
+  function(generate_zip zip_out bin_file_list deps)
```

Comment:
Maybe generate_dfu_zip, so readers don't think this is just about a generic zip file.

Reply:
agree